server/flyway/sql/V34__migrate_fam_role_assignment_to_admin_table.sql

```diff
@@ -0,0 +1,15 @@
+-- migrate the fam roles to fam admin management table
+-- first select all fam roles
+-- and then select role_id and application_id this role is admin of, based on if the role name contains the application_name
+-- -- for example, role name: FAM_ACCESS_ADMIN contains application name: FAM
+-- --              role name: FOM_DEV_ACCESS_ADMIN contains application name: FOM_DEV
+-- select the user_id and application_id the user is admin of, insert into fam_application_admin
+INSERT INTO app_fam.fam_application_admin (user_id, application_id, create_user, create_date)
+SELECT user_role_xref.user_id, application.application_id, CURRENT_USER, CURRENT_DATE
+FROM app_fam.fam_role role
+JOIN app_fam.fam_application application
+    ON role.role_name LIKE '%' || application.application_name || '%'
+JOIN app_fam.fam_user_role_xref user_role_xref
+    ON role.role_id = user_role_xref.role_id
+WHERE role.application_id=1;
```

Comment:
(since you mentioned) do you feel it is safer to use sub-query to replace this `application_id=1` for all environment `application_id = (sub-query ... where application_name = 'FAM')`? 

Also, after migration, maybe we like to have second script to remove this user from FAM_ADMIN (`('GATITEBI')`) record and add one more person ('CMENG'), probably need to add into fam_user if not exists?
Not sure what @basilv think.

Reply:
Ian, there's strictly speaking no need for a migration, we should have already removed Gbola's access and added Catherine's access as fam admin in the various environments, which will then be mirrored in the new table.

Reply:
Ah, yes @basilv , I was too focused into sql and forgot the easy way. Thanks for mentioning that :)

Reply:
I was only wondering maybe use sub-query would be safer. But no need, since you already make sure the id.